platform/web-security/src/main/resources/webapp/messages/messages.properties

```diff
@@ -212,3 +213,11 @@ email.password=Password
 
 email.onetime_body=Someone has opened a one time token request for your account. If it was you, click the following link to login.
 email.onetime_footer=If you did not one time token please ignore this message.
+
+email.password_reminder_msg_p1=Password for account
```

Comment:
Instead of splitting a message into parts like this, create one message that uses {} for params